splitio/engine/evaluator.py

```diff
@@ -43,51 +43,51 @@ def _evaluate_treatment(self, feature, matching_key, bucketing_key, attributes,
         :param attributes: An optional dictionary of attributes
         :type attributes: dict
 
-        :param split: Split object
+        :param feature_flag: Split object
```

Comment:
Double checking, is `Split` a type or should it also be changed?

Reply:
It is a class, I put the task to change it in "Could" list